data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageMain.java

```diff
@@ -302,16 +233,16 @@ else if (nodeSelExpr.getChild(0).getType() == HiveParser.TOK_ALLCOLREF)
 	private void handleTokTabOrColOrElse(ASTNode nodeSelExpr) {
 		String col = null;
 		String colName = null;
-		String table_alias = null;
+		String tableAlias = null;
 		String label = null;
 		String constant = null;
 		String constant_alias= null;
 		if (nodeSelExpr.getChild(0).getChild(0) != null
 				&& nodeSelExpr.getChild(0).getChild(0).getType() == HiveParser.TOK_TABLE_OR_COL) {
-			table_alias = nodeSelExpr.getChild(0).getChild(0).getChild(0).toString();
+			tableAlias = nodeSelExpr.getChild(0).getChild(0).getChild(0).toString();
 			 {
 			col = colName = nodeSelExpr.getChild(0).getChild(1).toString();
-			col = col + "->" + table_alias;
+			col = col + "->" + tableAlias;
 		    if(nodeSelExpr.getChild(1)!=null) label=nodeSelExpr.getChild(1).toString();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00122)